docs/recipes/handlebars.md

```diff
@@ -58,32 +97,39 @@ gulp.task('serve', ['styles', 'templates', 'fonts'], () => {
 Edit your `serve` task so that (a) editing an `.hbs` file triggers the `templates` task, and (b) the browser is reloaded whenever a `.js` file is generated in `.tmp/templates`:
 
 ```diff
- gulp.task('serve', ['styles', 'templates', 'fonts'], () => {
+gulp.task('serve', ['styles', 'templates', 'partials', 'scripts', 'fonts'], () => {
```

Comment:
The original `gulpfile.js` has changed (that is my fault for not merging this earlier, sorry 😓). Could you please update this task to reflect the changes? So instead of adding `partials` and `scripts` to the task itself, add it to the [line below](https://github.com/yeoman/generator-webapp/blob/master/app/templates/gulpfile.js#L97).